MacOS Mapping (#27064)
demisto/content: Packs/MacOS/README.md

# MacOS
This pack includes Cortex XSIAM content. 
MacOS Ventura is the current supported OS for XSIAM.
<~XSIAM>
## Configuration on Server Side
You need to configure MacOS to forward Syslog messages.

Open your MacOS device and follow these step:
1. Open a Terminal on the relevant device.
2. Open the syslog config file with a vi editor, write the command **vi /etc/syslog.conf**.
3. Type **i** in order to enter editing mode.
4. Write the following like as a new line in the syslog config file- \*.\* \@"\<IP address\>"\:"\<Port\>".

> 5. Write **:wq** to save and exit the file.

Reviewer: ```suggestion
5. Type **:wq** to save and exit the file.

```

> 6. Restart the syslogd daemon by writing the command **sudo launchctl stop com.apple.syslogd** and than **sudo launchctl start com.apple.syslogd**.

Reviewer: ```suggestion
6. Restart the syslogd daemon by typing the following commands:
      **sudo launchctl stop com.apple.syslogd**
      **sudo launchctl start com.apple.syslogd**.

```
